billing_adding_users_admin_assisted.md

```diff
@@ -1,5 +1,5 @@
 # M-RETS Administrator Assisted User Edits
 
-In the event that Users associated with the Organization with Manage Organization permission are no longer available to access M-RETS; M-RETS will require an affidavit on company letterhead before user permissions are changed or new users added to the Organization  The affidavit will explain that the user is no longer employed with the Organization and must be signed by a member of the Organization's autherized to make contracting decisions. 
+In the event that Users associated with the Organization with Manage Organization permission are no longer available to access M-RETS, M-RETS will require an affidavit on company letterhead before user permissions are changed or new users added to the Organization  The affidavit will explain that the user is no longer employed with the Organization and must be signed by a member of the Organization's authorized to make contracting decisions. 
```

Comment:
Just missing a period but looks good otherwise: `...or new users added to the Organization.  The affidavit will explain...`